### PR TITLE
chore: Add encode-decode

### PR DESCRIPTION
**Pull request by kingthorin**
- README.md > A short intro doc.
- double-spacer.js > The example script from: https://www.zaproxy.org/blog/2021-03-29-zap-2-10-0-features/
- build.gradle.kts > Added scriptType.
- CHANGELOG..md > Added note about the addition(s).

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: encode-decode/double-spacer.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Process the input value and return the encoded/decoded/hashed etc. value
+ *
+ * Use EncodeDecodeResult.withError("Error Description") to provide an error description
+ * inside the result view
+ *
+ * @param {String} value - The input value
+ * @returns {EncodeDecodeResult} - The value that was encoded/decoded/hashed etc.
+ */
```
Comment (thc202) [inline encode-decode/double-spacer.js:11]: Remove.

**Comment by kingthorin** [comment]
Done

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
Oops, damnit.

**Comment by kingthorin** [comment]
Anyone know why it's taking LGTM hours to analyze 3 lines?

**Comment by thc202** [comment]
It's good now.

**Comment by kingthorin** [comment]
Thanks!

**Pull request merged**